packages/web3torrent/src/library/web3torrent-lib.ts

```diff
@@ -137,7 +137,7 @@ export default class WebTorrentPaidStreamingClient extends WebTorrent {
     log.info('> Cancelling download. Closing payment channels, and then removing torrent');
     const torrent = this.torrents.find(t => t.infoHash === infoHash);
     if (torrent) {
-      await this.closeDownloadingChannels(torrent);
+      await this.closeChannels(torrent, true);
```

Comment:
Just to make sure I understand correctly. When the leecher triggers cancel, the leecher's seeding channels will be cancelled. Is that the behavior that we want?

Reply:
Yes. I believe so, because when the torrent is cancelled, it's gets removed from the client, which makes it impossible to seed or leech. 
This way, the peer makes an attempt to close the channel before cancelling the download.